backend/src/main/java/com/github/abcdgames/backend/player/Player.java

```diff
@@ -0,0 +1,23 @@
+package com.github.abcdgames.backend.player;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity(name = "player")
+public class Player {
+    @Id
+    @Column(name = "id", nullable = false)
+    @GeneratedValue(strategy = GenerationType.UUID)
```

Comment:
Wirklich UUID? Nicht einfach von 1 anfangen zu zählen?

Reply:
Was wäre ein Vorteil/Nachteil des jeweiligen ? Mir fällt keiner ein ? Das einzige was mir als Nachteil einfallen würde es inkremental hochzuzählen wäre wenn die api nicht ordentlich geschützt wäre und jemand wahllos mit id 1,2,3 Anfragen ans Backend macht. Bei einer uuid wäre das schon schwieriger.

Reply:
Ich bevorzuge UUID, weil wie david sagt wenn wir hochzählen kann man einfach +1 oder -1 abfragen und viel leichter die Id erraten. Was wenn man alles richtig macht vielleicht nicht sooo schlimm ist. Aber man kann auch sehr gut abschätzen wie viele Daten es gibt. Und was würde gegen uuid sprechen?
🤷

Reply:
Ich habe es damals so gelernt das die id mit 1 anfängt hochgezählt wird🤷‍♀️ mit was arbeitest du denn @Dskottke ? Kannst nicht mal in eine muster Datenbank reinschauen bei euch?